sql/moz-fx-data-shared-prod/telemetry/unified_metrics/view.sql

```diff
@@ -6,3 +6,5 @@ SELECT
   `mozfun.norm.browser_version_info`(app_version) AS browser_version_info
 FROM
   `moz-fx-data-shared-prod.telemetry_derived.unified_metrics_v1`
+WHERE
+  normalized_app_name != 'Focus Android Glean'
```

Comment:
Hi Lucia, can you verify these things for me? 

First, IF you happen to be currently running query against `unified_metrics_v1`, for example:

```
SELECT
    submission_date,
    COUNTIF(normalized_app_name != 'Firefox Desktop') AS cdou_mobile
FROM telemetry.unified_metrics_v1
WHERE days_since_seen = 0
AND submission_date >= DATE(2022,1,1)
GROUP BY 1
ORDER BY 1
```

THEN you would now be counting focus android clients on BOTH glean and legacy. In other words, if you were running this query before and after this PR, your results will change.

As far as I can tell, the official dashboards that track the KPI in looker use the *view*, which should NOT change: 
https://github.com/mozilla/looker-hub/blob/main/kpi/views/unified_metrics.view.lkml#L254

Finally, the view that would be found in the redash table browser looks to be called `telemetry.unified_metrics`, so users who queried the table based on that should also NOT have their results change from their queries.

Reply:
Hi Leif,

Yes to your first question. After merging this change and backfilling, if you query `telemetry_derived.unified_metrics_v1` without filtering out `Focus Android Glean` the result will be counting BOTH Glean and Telemetry.

About Looker, that's true, the definition in Looker is querying the view and there are no plans to change it for this change.

About Redash, for what I could also investigate it is pointing to the view:
<img width="433" alt="image" src="https://user-images.githubusercontent.com/30448600/200003659-bd971049-ec32-41cb-a1a6-0ee77368f1ea.png">

Reply:
> Yes to your first question. After merging this change and backfilling, if you query `telemetry_derived.unified_metrics_v1` without filtering out `Focus Android Glean` the result will be counting BOTH Glean and Telemetry.

Note that this is the expected behaviour, as proposed, discussed and approved in [this proposal](https://docs.google.com/document/d/1ZM2gt4dSj5scOcdgQO5ca2vpP7IJBOyVj6HuFfGLd4s/edit#). Consumers should be querying the view, as that's the interface to the data. With that said, the change will need to be communicated so that folks are prepared for this change if they are querying `unified_metrics_v1`.